MyWeather/Networking/Weather.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Weather.swift
+//  MyWeather
+//
+//  Created by Luis Gonzalez on 11/13/18.
+//  Copyright © 2018 Luis Gonzalez Jr. All rights reserved.
+//
+
+import Foundation
+
+struct Weather: Decodable {
+    let coord: [String: Double]
+    var weather: [[String:Int]] = []
+    let main: [String: Double]
+    let wind: [String: Double]
+    let sys: [String: Double]?
+    let humidity: Int
+    let visibility: Int
+
+
+}
```

Comment:
This is what your model should look like. 
You'll need to create objects for every dictionary you would like to use. 
Below is a model for the general response object, `WeatherResponse`. Since we only want `main`, then that the only property we will define for now.  `main` will be our true Weather object. In `Weather` you should give your model's properties better names, the API does a poor job at naming. Naming is always an issue in programing, but good naming of variables goes a long way. The property mappings to better names can be done by defining a `CodingKey` enum with the property name mappings. Let me know if this makes sense.

```suggestion
import Foundation

struct WeatherResponse: Codable {
    var main: Weather
}

struct Weather: Codable {
    enum CodingKeys: String, CodingKey {
        case currentTemperature = "temp"
        case pressure
        case humidity
        case minTemperature = "temp_min"
        case maxTemperature = "temp_max"
    }

    var currentTemperature: Double
    var pressure: Double
    var humidity: Double
    var minTemperature: Double
    var maxTemperature: Double
}
```

Reply:
I understand it. Im access the dictionary and being able to specify the type and name i want to use to access the data.

Reply:
✅